src/deadline/client/cli/__init__.py

```diff
@@ -4,3 +4,18 @@
 
 from . import deadline_dev_gui_main
 from ._deadline_cli import main
+
+# New groups must be imported here to get added to the main CLI as subcommands
```

Comment:
We were keeping them out of here to exclude them from the public interface of the library. Putting them here means that `deadline.client.cli.bundle_group` is a public interface instead of it being in internal implementation detail. What does importing them here provide for this PR?

To better understand what I mean about the public interface, read this section of the development guidelines: https://github.com/aws-deadline/deadline-cloud/blob/mainline/DEVELOPMENT.md#public-contracts